### ISSUE TITLE
subscription won't work with sca after I update to newer stripe api version

### ISSUE THREAD

**Issue by Reklen** [pay-rails/pay#195]
I switched to api_version '2020-08-27' since they added SCA which allow us to pay with 3D Secure, but unfortunately subscription won't pass with SCA since I get this error
```
NoMethodError (undefined method `subscriptions' for #<Stripe::Customer:0x00007f84c3dfc8a8>
Did you mean?  description):

app/controllers/subscriptions_controller.rb:23:in `create'
```
 
and here's Create method in subscriptions_controller.rb

```
19:   def create
20:     current_user.assign_attributes(subscription_params)
21:     @plan = Plan.without_free.find(current_user.plan) # Get the Stripe or Braintree specific ID
22:     processor_id = @plan.processor_id(current_user.processor)
23:     current_user.subscribe(plan: processor_id, trial_period_days: @plan.trial_period_days)
24:     redirect_to root_path, notice: t(".created")
25:   rescue Pay::ActionRequired => e
26:     redirect_to pay.payment_path(e.payment.id)
27:   rescue Pay::Error => e
28:     flash[:alert] = e.message
29:     render :new
30:   end
```

**Comment by excid3** [comment]
I just released `v2.2.0` yesterday that uses `Stripe::Subscription.create` instead of `customer.subscription.create` to fix this. 👍

**Comment by Reklen** [comment]
Thank you @excid3 awesome work 👍
